how-to/migrate_models.rst

```diff
@@ -0,0 +1,79 @@
+JAAS: Model Migration
+=====================
+
+In many scenarios it is necessary to migrate models from existing Juju controllers
+to an environment that includes JAAS.
+
+This process also allows you to migrate a model away from JAAS.
+
+Prerequisites
+-------------
+
+- A standalone Juju controller with a model (optionally with a running application).
+- A basic understanding of Juju model migrations, see the `docs <https://juju.is/docs/juju/manage-models>`__.
+- A running JAAS, see the :doc:`tutorial <../tutorial/deploy_jaas_microk8s>`.
+- Administrator permissions for JAAS, so our :doc:`how-to <./bootstrap_permissions>`.
+
+1. Create a new Juju controller
+-------------------------------
+
+This is only necessary if you have a Juju controller that does not have the ``login-token-refresh-url`` config option set to point 
+at a running JIMM instance. Use the following command to check if your controller is configured.
```

Comment:
Our language is getting a bit odd, we're witching between JIMM / JAAS, I get why but we need to make this feel simpler eventually.

Reply:
I agree. I try to JAAS when referring to the system and JIMM when referring to only the single component in JAAS, definitely not always successfully. I'm not opposed to just using one choice everywhere, probably JAAS.